woocommerce-gateway-stripe.php

```diff
@@ -5,7 +5,7 @@
  * Description: Take credit card payments on your store using Stripe.
  * Author: WooCommerce
  * Author URI: https://woocommerce.com/
- * Version: 4.1.14
+ * Version: 4.2
```

Comment:
Minor nitpick but why would we be bumping the version number prior to a release?

Reply:
Can a nitpick of a PR with a single line change be minor? 🤔 🤣 

I am preparing the branch structure for SCA, but there was an `sca` branch already (now named `stripe-initial-scrapped`) and I was seeing some conflicts like this:

> error: update_ref failed for ref 'refs/remotes/origin/sca/version-bump': cannot lock ref 'refs/remotes/origin/sca/version-bump': 'refs/remotes/origin/sca' exists; cannot create 'refs/remotes/origin/sca/version-bump'

I created this PR to ensure that that:

1. The new branches are in place.
2. I can create a PR that merges into a branch that does not have a PR yet.

Seems to work so far :)

P.S. There is a reason the PR is not marked as waiting for review :)

Reply:
> P.S. There is a reason the PR is not marked as waiting for review.
Fair enough, it just came through my inbox and I was curious, I didn't open a review on it just a comment 😄